### ISSUE TITLE
pVACtools examples tutorials

### ISSUE THREAD

**Issue by ne1al** [griffithlab/VAtools#31]
Hello!
I am trying to find the tutorials to generate files and how to use them for run pvactools here:
http://www.genomedata.org/pvactools-examples/
The files: http://www.genomedata.org/pvactools-examples/annotated.expression.vcf.gz
http://www.genomedata.org/pvactools-examples/annotated.phased.vcf.gz
Could you please help?
Thank you

**Comment by susannasiebert** [comment]
I'm not familiar with those files or how they were generated. They might be related to this tutorial: https://pmbio.org/module-08-immune/0008/01/01/Intro_to_Immunogenomic_Analysis/ although I don't see any instructions on how precisely they were generated. My best guess is that they were generate following all of the upstream steps in the different pmbio.org courses.

**Comment by malachig** [comment]
I think @huimingx knows the most about this module of the PMBIO course.  Is that what you are attempting to follow @ne1al ?

If so, this issue would probably make more sense posted to that repo: https://github.com/griffithlab/pmbio.org

**Comment by huimingx** [comment]
I believe those files were generated for the purposes of our pvacviz online demo and got uploaded to genomedata.org for book keeping purposes. They are generated from the same data (HCC1395) as used in the tutorials on pmbio.org. I think pmbio.org specifies the steps up to obtaining a vep annotated somatic vcf (Modules01-05) as well as obtaining expression data (Module-06). The coverage and expression data can then be added to the somatic vcf using the steps specified at https://pvactools.readthedocs.io/en/latest/pvacseq/input_file_prep.html. 

As for the phased vcf, the input data can also be downloaded from pmbio.org and the steps I followed to generate the annotated.phase.vcf.gz are here: https://pvactools.readthedocs.io/en/latest/pvacseq/input_file_prep/proximal_vcf.html. 

Unfortunately, the workflow currently described at https://pmbio.org/module-08-immune/0008/02/01/Personalized_Cancer_Vaccine_Design/ focuses on getting the appropriate AWS setup to run pvactools but assumes you have a proper vcf generated prior to following those steps. Until we have that updated, the steps I outlined should be able to get you started on generating those files. Let me know if you run into any further issues.
